ADAL/src/public/ADTokenCacheItem.h

```diff
@@ -25,6 +25,20 @@
     Objects of this class are used in the key-based token cache store.
     See the key extraction function for details on how the keys are constructed. */
 @interface ADTokenCacheItem : NSObject<NSCopying , NSSecureCoding>
+{
+    NSUInteger _hash;
+    NSString* _resource;
+    NSString* _authority;
+    NSString* _clientId;
+    NSString* _familyId;
```

Comment:
where is familyId being set? also, we are using "foci" as a value in the server response.